src/ConnectedKubernetes/ConnectedKubernetes.Autorest/docs/Set-AzConnectedKubernetes.md

```diff
@@ -88,7 +144,7 @@ This command disable gateway feature of a connected kubernetes.
 
 ### Example 2: Enable gateway feature of connected kubernetes.
```

Comment:
```suggestion
### Example 2: Enable gateway for a connected Kubernetes cluster
```

Think writing it like this matches our public docs better

(here and elsewhere)